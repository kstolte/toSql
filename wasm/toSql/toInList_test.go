package toSql

import (
	"testing"

	"github.com/gkampitakis/go-snaps/snaps"
)

func Test_SimpleList(t *testing.T) {
	in := `a
b
c
d
e
`
	got, err := ToInList(in)
	if err != nil {
		t.Errorf("received error %d", err)
	}
	if got == "" {
		t.Error("encounterd blank string return value")
	}
	snaps.MatchSnapshot(t, got)

}
