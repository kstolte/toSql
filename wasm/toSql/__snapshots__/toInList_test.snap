
[Test_SimpleList - 1]
IN ('a',
'b',
'c',
'd',
'e'
)

---
